static/main.js

```diff
@@ -182,7 +180,13 @@ socket.on("work_finish", function(message) {  // Receive and download results
             document.body.removeChild(a);
         });
     }
+    if (processing_in_progress) {
+        let init_info_1 = document.getElementById("init_info_1");
+        init_info_1.innerHTML = "";
```

Comment:
Using = operand overwrites current value, why do you have to equal it to "" if next line you equal it to something else